crates/types/src/invocation.rs

```diff
@@ -1247,3 +1369,67 @@ mod mocks {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    mod invocation_response {
+        use super::*;
+
+        #[allow(non_camel_case_types)]
+        #[derive(Debug, Clone, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
+        pub struct InvocationResponse_1_2 {
+            pub id: InvocationId,
+            pub entry_index: EntryIndex,
+            pub result: ResponseResult,
+        }
+
+        #[test]
+        fn front_compat() {
```

Comment:
What is `front_compat`? Do you mean forward compatibility? If yes, then let's spell it out.